examples/03_App.ipynb

```diff
@@ -153,6 +153,46 @@
     "}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Custom `ElkNode` css classes\n",
+    "\n",
+    "Custom css classes can be passed to `ElkNode`, `ElkPort`, and `ElkEdge`. The `XELK`\n",
+    "transformer has a nested dictionary where the keys are nodes in the source networkx\n",
+    "graph and the values are another dictionary with Elk types as keys and values are sets\n",
+    "of strings. This set gets applied to the Elk types that are generated for the given\n",
+    "networkx node."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipyelk.diagram.elk_model import ElkEdge, ElkNode\n",
+    "\n",
+    "elk_app.transformer.css_classes = {\n",
+    "    \"n1\": {\n",
+    "        ElkNode: set([\"example-custom-class\"]),\n",
```

Comment:
As discussed, using classes as keys kinds freaks me out...

Reply:
Created new issue to track https://github.com/jupyrdf/ipyelk/issues/31.